h/static/scripts/directive/annotation.coffee

```diff
@@ -202,6 +208,10 @@ AnnotationController = [
       switch @action
         when 'create'
           updateDomainModel(model, @annotation)
+
+          if group.focusedGroup().hashid != '__public__'
```

Comment:
We could also just use the word `id` instead of `hashid` wherever the hashid appears in the client code, probably true that it doesn't need to know that it's a hashid.